src/GridProEditColumn.tsx

```diff
@@ -41,11 +42,79 @@ export type GridProEditColumnProps<TItem> = Partial<
     renderer?: ComponentType<GridBodyReactRendererProps<TItem>> | null;
   }>;
 
+type ReactBodyRenderer<TItem> = ComponentType<GridBodyReactRendererProps<TItem>> & {
+  __wrapperRenderer?: ReactBodyRenderer<TItem>;
+};
+
+type EditColumnRendererRoot = HTMLElement & { __editColumnRenderer?: GridBodyRenderer };
+
+type ClearFunction = (arg0: HTMLElement & { _content: EditColumnRendererRoot }) => void;
+
+/**
+ * Wraps a React renderer function to render empty when requested
+ *
+ * @returns
+ */
+function editColumnReactRenderer<TItem>(reactBodyRenderer?: ReactBodyRenderer<TItem> | null) {
+  if (!reactBodyRenderer) {
+    return undefined;
+  }
+
+  reactBodyRenderer.__wrapperRenderer ||= (props) => {
```

Comment:
This is currently shown as `Anonymous` in React dev tools component tree:

<img width="272" alt="Screenshot 2024-01-09 at 14 55 20" src="https://github.com/vaadin/react-components/assets/10589913/3c371e38-2132-46b6-9e8f-664e8213ed9d">

Consider using `displayName` to make it more clear e.g. something like this:

```js
  reactBodyRenderer.__wrapperRenderer.displayName = 'GridProEditColumnRenderer';
```

Reply:
Actually, it's enough to change the arrow function to `function GridProEditColumnRenderer (props)`.

Reply:
Done